Week-1/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+
+<head>
+  <title>Assignment Week 1</title>
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+</head>
+<body>
+  <div class="page">
+    <div class="sidebar">
+      <a href="#"><i class="fa fa-fw fa-bars"></i></a>
+      <a href="#"><i class="fa fa-fw fa-home"></i></a>
+      <a href="#"><i class="fa fa-fw fa-search"></i></a>
+      <a href="#"><i class="fa fa-fw fa-volume-up"></i></a>
+      <a href="#"><i class="fa fa-fw fa-user"></i></a>
+      <a href="#"><i class="fa fa-fw fa-spotify"></i></a>
+      <a href="#"><i class="fa fa-fw fa-cog"></i></a>
+      <a href="#"><i class="fa fa-fw fa-soundcloud"></i></a>
+    </div>
+    <div class="content">
+      <div class="first-ele">
+        <div class="image">
+          <img src="./cattyboard.jpg" width="150" height="150" class="center" />
+        </div>
+        <div class="first-content">
+          <h3 style="color: white">
+            CattyBoard Top 100 Single Charts(11.06.36)
+          </h3>
+          <p style="margin-top: -10px; color: white">Unknown Artist</p>
+          <p style="margin-top: -10px; color: white">2016 Charts 100 songs</p>
+          <div class="buttons">
+            <span style="padding-right: 10px"
+              ><i class="fa fa-fw fa-play"></i>play</span
+            >
+            <span style="padding-right: 10px"
+              ><i class="fa fa-fw fa-plus"></i>Add to</span
+            >
+            <span style="padding-right: 10px"
+              ><i class="fa fa-fw fa-ellipsis-h"></i>More</span
+            >
+          </div>
+        </div>
+      </div>
+      <div class="second-ele">
+        <table>
+          <tr>
+            <th>1.One Dance</th>
+            <th>Crake feat CatKid &Cyla</th>
+            <th>2:54</th>
+            <th><a href="#">CATY CLOUD SYNC</a></th>
+          </tr>
+          <tr>
+            <th>2.Panda</th>
+            <th>Catte</th>
+            <th>4:06</th>
+            <th><a href="#">CATY CLOUD SYNC</a></th>
+          </tr>
+          <tr>
+            <th>3.Can't Stop the Feeling!</th>
+            <th>Catin Cimberlake</th>
+            <th>3:56</th>
+            <th><a href="#">CATY CLOUD SYNC</a></th>
+          </tr>
+          <tr>
+            <th>4.Work From Home</th>
+            <th>Cat Harmony feat Colla</th>
+            <th>3:34</th>
+            <th><a href="#">CATY CLOUD SYNC</a></th>
+          </tr>
+        </table>
+      </div>
+      <div class="third-ele">
+        <div class="image2">
+          <img src="./cattyboard.jpg" width="150" height="150" class="center" />
```

Comment:
alt tag

Reply:
in my screen it is not scrolling after viewport